dockstore-webservice/src/test/java/io/dockstore/webservice/languages/CWLHandlerTest.java

```diff
@@ -101,4 +105,58 @@ public void testURLFromEntry() {
         Assert.assertEquals("https://www.dockstore.org/containers/registry.hub.docker.com/foo/bar", handler.getURLFromEntry("foo/bar", toolDAO));
         Assert.assertEquals("https://www.dockstore.org/containers/registry.hub.docker.com/foo/bar", handler.getURLFromEntry("foo/bar:1", toolDAO));
     }
+
+    @Test
+    public void testGetContentWithMalformedDescriptors() throws IOException {
+        CWLHandler cwlHandler = new CWLHandler();
+
+        // create and mock parameters for getContent()
+        final Set<SourceFile> emptySet = Collections.emptySet();
+        final ToolDAO toolDAO = Mockito.mock(ToolDAO.class);
+        when(toolDAO.findAllByPath(Mockito.anyString(), Mockito.anyBoolean())).thenReturn(null);
+
+        // expect parsing error
+        File cwlFile = new File(ResourceHelpers.resourceFilePath("brokenCWL.cwl"));
+        try {
+            cwlHandler.getContent("/brokenCWL.cwl", FileUtils.readFileToString(cwlFile, StandardCharsets.UTF_8), emptySet,
+                LanguageHandlerInterface.Type.TOOLS, toolDAO);
```

Comment:
put `Assert.fail("blah blah")` under this to be certain it actually had an exception. same with other places

Reply:
Fixed in both [WDL](https://github.com/dockstore/dockstore/blob/c5277c33e2a3bf271f3be4e33b742fb646b1a076/dockstore-webservice/src/test/java/io/dockstore/webservice/languages/WDLHandlerTest.java#L171) and [CWL](https://github.com/dockstore/dockstore/blob/c5277c33e2a3bf271f3be4e33b742fb646b1a076/dockstore-webservice/src/test/java/io/dockstore/webservice/languages/CWLHandlerTest.java#L124) test files.